packages/safe-apps-test-app/src/App.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { Spinner, Heading, SegmentedControl } from 'evergreen-ui';
+import SafeAppsSDK, { SafeInfo } from '@gnosis.pm/safe-apps-sdk';
+import { AppTabs } from './types';
+import Main from './tabs/Main';
+import RpcCalls from './tabs/RpcCalls';
+
+const Container = styled.div`
+  padding: 24px;
+  margin-bottom: 2rem;
+  width: 100%;
+  max-width: 480px;
+
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-column-gap: 1rem;
+  grid-row-gap: 1rem;
+`;
+
+const tabs = [
+  {
+    value: '0',
+    label: 'main',
+  },
+  { value: '1', label: 'RPC' },
+];
+
+const SDK = new SafeAppsSDK();
+
+const App = (): React.ReactElement => {
+  const [safeInfo, setSafeInfo] = useState<SafeInfo | undefined>();
+
+  useEffect(() => {
+    navigator.mediaDevices
+      .getUserMedia({ video: true, audio: true })
+      .then(function (stream) {
+        console.log('fine');
+      })
+      .catch(function (err) {
+        console.error(err);
+      });
+
+    async function loadSafeInfo() {
+      const safuInfo = await SDK.safe.getInfo()
```

Comment:
Minor typo here right? `safuInfo` -> `safeInfo `

Reply:
Not really a typo, it comes from "funds are safu" meme

Reply:
https://www.urbandictionary.com/define.php?term=Safu

Reply:
if you find it v confusing I can change it no problem

Reply:
😂 Ok I think Dani will be fine with your explanation

Reply:
no idea about this